pkg/actors/actors.go

```diff
@@ -347,9 +347,14 @@ func (a *actorsRuntime) GetState(ctx context.Context, req *GetStateRequest) (*St
 	if a.store == nil {
 		return nil, errors.New("actors: state store does not exist or incorrectly configured")
 	}
+
+	partitionKey := a.constructCompositeKey(a.config.AppID, req.ActorType, req.ActorID)
+	metadata := map[string]string{"partitionkey": partitionKey}
```

Comment:
This is especially for actors. I think we need to update api or some docs?

Reply:
Yes, this https://github.com/dapr/docs/blob/master/howto/setup-state-store/setup-azure-cosmosdb.md.  I'll update that after this is finalized.